packages/ui-app/src/AddressSummary.tsx

```diff
@@ -70,58 +84,92 @@ class AddressSummary extends React.PureComponent<Props> {
   }
 
   protected renderAddress () {
-    const { defaultName, isShort = true, value } = this.props;
+    const { isShort = true, value } = this.props;
 
-    if (!value) {
-      return null;
-    }
-
-    const address = value.toString();
-    const name = getAddrName(address, false, defaultName);
+    const address = value
+      ? value.toString()
+      : DEFAULT_ADDR;
 
     return (
-      <div className='ui--AddressSummary-data'>
-        <div className='ui--AddressSummary-name'>
-          {name}
-        </div>
         <div className='ui--AddressSummary-accountId'>
           {
             isShort
               ? toShortAddress(address)
               : value
           }
         </div>
-      </div>
     );
   }
 
-  protected renderAccountId () {
-    const { accounts_idAndIndex = [], defaultName, isShort = true, value } = this.props;
-    const [_accountId, accountIndex] = accounts_idAndIndex;
+  protected renderName () {
+    const { accounts_idAndIndex = [], defaultName, isEditable, value } = this.props;
+    const { isEditing } = this.state;
+    const [_accountId] = accounts_idAndIndex;
     const accountId = _accountId || value;
 
-    if (!accountId && accountIndex) {
-      return null;
-    }
-
     const address = accountId
       ? accountId.toString()
       : DEFAULT_ADDR;
-    const name = getAddrName(address, false, defaultName);
 
-    return (
-      <div className='ui--AddressSummary-data'>
-        <div className='ui--AddressSummary-name'>
+    const name = getAddrName(address, false, defaultName) || '';
+
+    let className = 'ui--AddressSummary-name';
```

Comment:
can just use `ui-app/util/classes` - it removes blanks and concats

Reply:
oh that's nice, hadn't noticed it before.